src/testFixtures/java/uk/gov/di/ipv/cri/common/library/client/StubClient.java

```diff
@@ -54,21 +58,12 @@ public PrivateKeyJWT generateClientAssertion(String issuer, String audience)
     }
 
     private ECKey getEcPrivateKey() throws ParseException {
-        return ECKey.parse(getParameterValueByAbsoluteName());
-    }
-
-    private String getParameterValueByAbsoluteName() {
-        try {
-            GetParameterRequest parameterRequest =
-                    GetParameterRequest.builder()
-                            .name("/test-resources/ipv-core-stub-aws-headless/privateSigningKey")
-                            .build();
-
-            GetParameterResponse parameterResponse = ssmClient.getParameter(parameterRequest);
-            return parameterResponse.parameter().value();
-        } catch (SsmException e) {
-            e.getMessage();
-            throw e;
-        }
+        return ECKey.parse(
+                ssmHelper
+                        .getParameterValueByName(
+                                "/test-resources/"
```

Comment:
maybe we shouldn't hard-code `/test-resources/` use this.testResourcesStackName just so this may be testable from a custom test-resources stack